challenges/classes.js

```diff
@@ -1,7 +1,30 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
-
+class CuboidMaker2{
+    constructor(length, width, height){
+        this.width = width;
+        this.height = height;
+        this.length = length;
+      }
+    volume(){
+        return this.length * this.width * this.height;
+    }
+    surfaceArea(){
+        return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
```

Comment:
please use brackets to mark the separation  between 2 operator. so, it's easier to read do something like

`return 2 * ((this.length * this.width) + (this.length * this.height) + (this.width * this.height));`